src/blocks-ext.js

```diff
@@ -45,6 +45,9 @@ BlockMorph.prototype.showHelp = async function() {
         metadata = isServiceURL ?
             await Services.getServiceMetadataFromURL(serviceName) :
             await Services.getServiceMetadata(serviceName);
+
+        const isFSService = !!metadata.servicePath;
```

Comment:
Hmm... It isn't obvious to me that this should be exposed in the metadata (which makes me a bit reluctant to have other features depend on it). Perhaps there is a better way to capture this information (ie, if it has official documentation) in the metadata and then use that here instead.

Reply:
agreed. I suspect there might be something better than `isOfficial` but I am not exactly sure what just yet.

My issue with `isOfficial` is that the server-side has been decoupling these further and further and there may be a more general term rather than `isOfficial` for 3rd party service hosts to designate that they have documentation. (Even in the current server, you can specify additional URLs to use for services and nest them under a dedicated category.)

Reply:
For example, you could make `services.c2stem.org` which could be nested under an arbitrary `c2stem` category. Here is an example services server written in Python: https://github.com/NetsBlox/Custom-Python-Services. 

In the new cloud, they can be edited using the CLI [here]( https://github.com/NetsBlox/cloud/blob/main/crates/cli/src/main.rs#L175-L216). Basically, they can be defined for individual users or class/groups. When they are registered, they require a URL and categories to be nested under. This allows them to show up in the NetsBlox browser for the affected users and they can implement basic request-reply (no message passing). If they need access to user information and message passing capabilities, they must be authorized by an admin of the NetsBlox cloud. This authorizes them to use some protected endpoints which should only be used by trusted servers as they can look up user states, send messages, etc.